packages/discord.js/src/structures/BaseChannel.js

```diff
@@ -155,14 +155,6 @@ class BaseChannel extends Base {
     return 'availableTags' in this;
   }
 
-  /**
-   * Indicates whether this channel is sendable.
-   * @returns {boolean}
-   */
-  isSendable() {
-    return 'send' in this;
-  }
```

Comment:
There should probably still be this helper on the structures themselves, as well as the send method which calls the channel manager.